docs/build.md

```diff
@@ -0,0 +1,66 @@
+# Electron Build Instructions
+
+## Common
+
+```bash
+git clone git@github.com:ubergrape/grape-electron.git
+cd grape-electron
+npm install
+npm run release
+```
+The result would be a installable (`dmg` and `app` or `msi` and `exe`) file in `./releases/` with version prefix that match version from `./app/package.json`
+
+## App signing
+
+### MacOS
+
+You need to sign release:
+
+- to upload it to MacOS AppStore
+- to create a signed release for a distribution without store
+- create and install all certificates from developer.apple.com
+
+  ![](./images/mac-certificates-screen.png)
+
+  http://stackoverflow.com/questions/29039462/which-certificate-should-i-use-to-sign-my-mac-os-x-application
+- Download provision profiles and put them into root:
+  - Electron_Chat_Dev.provisionprofile
+  - Electron_Chat_Prod.provisionprofile
+- Increment [build number](https://github.com/ubergrape/grape-electron/blob/master/app/package.json#L7) for each review upload (you can upload same version multiple times if review failed, but build should be uniq all the time, don't reset this number ever, even for new version)
+
+- Signed store release `npm run release -- --sign {teamId} --mas`
+- Signed distribution outside of store `npm run release -- --sign {teamId}`
+
+For signing you will need group of certificates in `Keychain Access`, described here: https://github.com/electron/electron/blob/master/docs/tutorial/mac-app-store-submission-guide.md#get-certificate
```

Comment:
To sign the app you first need to get a certificate from Apple as described here